index.js

```diff
@@ -21,6 +23,9 @@ module.exports = function (source) {
     const options = typeof optionsConfig === 'object' ?
         utils.getOptions(this) : utils.parseQuery(this.query);
 
+    if(options !== "undefined" && options !== null && options.verbose === true){
```

Comment:
```suggestion
    if (options && options.verbose) {
```

In this case the above will be enough, also make sure you keep spaces between `if` and `(` **and** `)` and `{`